src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -80,7 +80,7 @@ public ExternalXref[] GetValidateExternalXrefs()
             }).OrderBy(externalXref => externalXref.Uid).ToArray();
         }
 
-        public JToken TransformContent(ErrorBuilder errors, FilePath file)
+        public (JToken, JToken) TransformContent(ErrorBuilder errors, FilePath file)
```

Comment:
The models should be de-serialized from transformed models, if there are source info problem, we could fix them later by propagating source info into the de-serialized JToken.
```suggestion
        public JToken TransformContent(ErrorBuilder errors, FilePath file)
```

Reply:
OK